docs/TypeScript.md

```diff
@@ -393,6 +393,23 @@ However, there are a couple of suggestions to help improve this experience:
 - Make sure the `no-unused-vars` rule is enabled in [ESLint](https://eslint.org/docs/rules/no-unused-vars) and any imported plugin are actually being loaded.
 - Use a module such as [depcheck](https://www.npmjs.com/package/depcheck) or [npm-check](https://www.npmjs.com/package/npm-check) to verify plugin dependencies are being used somewhere in your project.
 
+## Code Completion In Vanilla JavaScript
+
+Vanilla JavaScript can use the published types to provide code completion (e.g. [Intellisense](https://code.visualstudio.com/docs/editor/intellisense)) by following the [TypeScript JSDoc Reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html).
```

Comment:
as far as I know jsdoc was around earlier than ts https://jsdoc.app/. not sure it's right to mention it's a ts thing

Reply:
upon a further look, it indeed seems this syntax is typescript specific, so we're good

Reply:
👍 also see https://twitter.com/swyx/status/1350427690814251010